library/gui/screen/src/main/java/org/quiltmc/qsl/screen/api/client/QuiltScreen.java

```diff
@@ -28,37 +28,40 @@
 import net.minecraft.client.gui.widget.ClickableWidget;
 import net.minecraft.client.render.item.ItemRenderer;
 
-import org.quiltmc.qsl.base.api.util.InjectedInterface;
-
 /**
- * Utility methods related to {@link Screen}s.
+ * An interface implemented by {@link Screen} through a mixin in order to expose QSL extensions and also provide utility methods.
  *
  * @see ScreenEvents
  */
 @Environment(EnvType.CLIENT)
-@InjectedInterface(Screen.class)
-public interface QuiltScreenExtensions {
+public interface QuiltScreen {
 	/**
-	 * Gets all of a screen's button widgets.
-	 * <p>
-	 * The provided list allows for addition and removal of buttons from the screen.
+	 * Gets all of the screen's button widgets.
+	 *
+	 * <p>The provided list allows for addition and removal of buttons from the screen.
```

Comment:
Please don't move those `<p>` tags to a start of a paragraph in an inline way.

Reply:
I've changed it back so it followed CONTRIBUTING.md; This resulted into some of the FAPI copy-paste being quiltified in the process